assets/output.html.erb

```diff
@@ -191,7 +190,7 @@
           idx,
           scrollY = window.scrollY;
         for (i = 0; i < boxPositions.length; i++) {
-          if (scrollY <= boxPositions[i]) {
+          if (scrollY <= boxPositions[i] - 1) {
```

Comment:
Subtracting one pixel is required to properly set up fixed file header after navigating to an anchor.